### PR TITLE
poc with attrs and navigation method abstraction

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**

### CODE REVIEW

**Comment by RonnyPfannschmidt** [comment]
@psav @mshriver this one needs a discussion and perhaps a internal card

File: navmazing/__init__.py [hunk 1]
```diff
@@ -147,22 +168,15 @@ class NavigateToObject(object):
     callable that will navigate to the prerequisite step on the other object.
     """
 
-    def __init__(self, other_obj, target, obj=None):
```
Comment (psav) [inline navmazing/__init__.py:150]: This is API changing - I'm sure this was in here for a reason - that we used it somewhere in CFME

Comment (RonnyPfannschmidt) [reply navmazing/__init__.py:150]: i recall this is just problematic code from the class based navigation, i beleive back then it tested well, as it was mainly used for passover - a crosschec should be done however

Comment (RonnyPfannschmidt) [reply navmazing/__init__.py:150]: i believe i vaguely recall finding a horrendous bug wrt naming too many things obj

**Review by quarckster — APPROVED** [review]
LGTM

**Comment by RonnyPfannschmidt** [comment]
closing as wont follow up for now

**Pull request closed without merging**